scripts/create_perf_json.py

```diff
@@ -1201,12 +1238,14 @@ def save_form(name, group, form, desc, locate, scale_unit, threshold,
                              'num_packages', 'num_cores', 'SYSTEM_TSC_FREQ',
                              'filter_tid', 'TSC', 'cha', 'config1',
                              'source_count', 'slots', 'thresh', 'has_pmem',
-                             'num_dies', 'num_cpus_online']:
+                             'num_dies', 'num_cpus_online', 'PEBS']:
```

Comment:
Probably not needed now.

Reply:
This check seems to be executed before function `extract_pebs_formula(formula)` is called. Therefore, we still need to add this term here to continue the execution.